.gitignore

```diff
@@ -3,4 +3,5 @@ node_modules
 coverage
 dist
 docs
-.env
\ No newline at end of file
+.env
+env.js
```

Comment:
This could break env.js of publisher and viewer demo packages

Reply:
Removed.